lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2016/data/ashrae_90_1_2016.prototype_inputs.json

```diff
@@ -229,7 +229,7 @@
       "pszac_operation_schedule_3": null,
       "pszac_oa_damper_schedule_3": null,
       "occ_sensing_exterior_lighting_power": 1459.5,
-      "occ_sensing_exterior_lighting_schedule": "RestaurantSitDown Exterior_lighting_schedule_b",
+      "occ_sensing_exterior_lighting_schedule": "RestaurantSitDown Exterior_lighting_schedule_b_2016",
```

Comment:
@leijerry888 - Is that a correct change? From [this PR](https://github.com/NREL/openstudio-standards/pull/1055/files) it shows that `Exterior_lighting_schedule_b` is correct.